src/app/rundown-view/factories/key-binding.factory.ts

```diff
@@ -12,6 +12,9 @@ import { RundownNavigationService } from '../../shared/services/rundown-navigati
 import { RundownCursor } from '../../core/models/rundown-cursor'
 import { Logger } from '../../core/abstractions/logger.service'
 
+const CAMERA_COLOR: string = '#005919'
+const REMOTE_COLOR: string = '#ac29a5'
```

Comment:
Here a comment about what colour it is would be nice.
So it would be `const CAMERA_COLOR: string = '#005919' // Green`

Reply:
Comments on color variables are destined to become outdated. When piece rendering is done, these values will be changed to css variables, such that this becomes something like:
```typescript
const CAMERA_COLOR: string = 'var(--camera-color)'
```